py3dtilers/IfcTiler/README.md

```diff
@@ -118,6 +118,28 @@ Project the features on another CRS. The `crs_in` flag allows to specify the inp
 ifc-tiler --file_path <path> --crs_in EPSG:3946 --crs_out EPSG:4171
 ```
 
+### Geometric error
```

Comment:
Same comment as for GeojsonTiler/Readme.md and all the Tilers/Readme.md below (I guess).